website/content/docs/agent-and-proxy/agent/process-supervisor.mdx

```diff
@@ -101,8 +101,7 @@ The top level `exec` block has the following configuration entries.
 
 - `restart_on_secret_changes` `(string: "always")` - Controls whether agent
   will restart the child process on secret changes. There are two types of
-  secret changes relevant to this configuration: a static secret update (on
```

Comment:
@JanMa was this line intentionally removed?

Reply:
No, this was by accident

Reply:
I think you have the same issue in #2078 (or a similar one) fwiw. :-)